### DISCUSSION TITLE
Filter a cell containing two different words (not side by side) [Jun 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2384] [Discord Threads]
> 🙋‍♂️ **vibrantium**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### One way I could think of is by using 2x `like` with an `and` in between.    
####    
#### You will have to replicate the same over API  
![image0](https://cdn.discordapp.com/attachments/983184575306039306/983240930981212230/Screenshot_2022-06-06_at_10.56.12_AM.png)  
#  
> 🙋‍♂️ **vibrantium**  
#### Thank you kindly!! I will try it, does the "Like" operator pick up words that include our search word? Say, if one of the words in the cell said "Amazonian" & we still search for "Amazon" will that cell still get picked up?  
#  
> 👨‍🦲 **dstala**  
#### yes  
#  
> 🙋‍♂️ **vibrantium**  
#### thank you for your help! I think a feature I'd like to request is to find a way so that if i use the "like" operator whenever I filter things, it picks up whole words instead of elements/parts of words? Like my example earlier, if I want to search "Amazon" in a cell full of other words but one cell spells it as "Amazonian" instead, that one cell shouldn't be selected?  
#  
> 👨‍🦲 **dstala**  
#### You can search for Amazon with a space suffix :: "Amazon "  
#  
> 🙋‍♂️ **vibrantium**  
#### oh okay that's good news!!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/983184575306039306)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
